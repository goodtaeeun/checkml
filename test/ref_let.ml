let ref_example () =
  let x = ref 42 in
  Printf.printf "The value of x is: %d\n" !x

let () = ref_example ()
