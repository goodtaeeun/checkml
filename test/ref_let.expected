File ref_let.ml, line 2, column 10
[31mError[0m: Don't use ref!

